### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by urish** [casser/intellij-es67#1]
First of all, thanks for creating this plugin! I installed it on my WebStorm, however, it still does not recognize the `await` keyword. Does the plugin require any special configuration?

Thanks!

**Comment by casser** [comment]
await is context depended keyword can you share code example where its not recognized ?

**Comment by urish** [comment]
Sorry, I meant to say that `async` is not recognized. Here is an example:

```
async function test() {
    return 1;
}

test().then(value => console.log(value));
```

WebStorm has an error marker that says `Expecting newline or semicolon` just after the `async` keyword.

**Comment by casser** [comment]
Did you change Javascript mode to `ECMAScript 6` ? 

Please check `Preferences -> Languages & Frameworks -> Javascript` to be sure that `Javascript language version` is set to `ECMAScript 6` , if still have an issue let me know your System, and exact WebStorm version

PS: I have tested plugin in **IntellijIDEA 14 Ultimate** on **Mac**, in theory it should work on WebStorm too but I have never tested directly

**Comment by urish** [comment]
Indeed, I have changed the language version to ECMAScript 6. Even tried switching back to 5.1 and then to 6 again, but to no avail.

Using WebStorm 9.0.3 (WS-139.1112), checked both on Linux and Windows 7. If you can give me some pointers how to debug this, I can look into it

**Comment by zazagag** [comment]
the same problem PhpStorm 8.0.3, Win8

**Comment by meenie** [comment]
I've just installed this plugin, turned on ES6 and it's still having issues with Annotations.

![image](https://cloud.githubusercontent.com/assets/1126819/6608024/e6cd07fe-c800-11e4-9d14-0f952e1da257.png)

**Comment by meenie** [comment]
@casser: Is there any sort of work around to get Annotations working?

**Comment by wmakeev** [comment]
I have the same issue with `async` keyword.

![async](https://www.evernote.com/l/AAEsUz-vK2ZPMq4VqDmsQ4KuDNhcoOjG7FUB/image.png)

MacOS X 10.10.3
WebStorm 9.0.3 (WS-139.1112)
Plugin version 1.0.1
ECMAScript 6 (selected)

**Comment by mlaccetti** [comment]
Similar issue with WebStorm 10.0.4 - OS X.

**Comment by meenie** [comment]
@casser: Just wanted to say that I've got this working now.  All I had to do was uninstall the plugin, restart, install the plugin, and then restart.  Thanks :)

**Comment by wmakeev** [comment]
@meenie what version of plugin and webstorm are you using?

**Comment by meenie** [comment]
Plugin: `1.2.0`
IDE: IntelliJ IDEA Ultimate `14.1.4`

**Comment by smashercosmo** [comment]
Also can't make it work (( 
Windows 7
Plugin: 1.2.0
WebStorm: 10.0.4

**Comment by osdio** [comment]
I also failed:
mac os 10.10.2
Plugin: 1.2.0
Webstorm: 10
